guides/prow_knative_setup.md

```diff
@@ -53,10 +53,10 @@ All Prow config files for running Prow jobs for Knative projects are under
    `//test/presubmit-tests.sh` working, and optionally `//hack/release.sh`
    working for automated nightly releases).
 
-1. Update [config_knative.yaml](../config/prow/config_knative.yaml) (usually,
+1. Update [config_knative.yaml](../config/prod/prow/config_knative.yaml) (usually,
    copy and update the existing configuration from another repository). Run
    `./hack/generate-configs.sh` to regenerate
-   [config/prow/jobs/config.yaml](../config/prow/jobs/config.yaml), otherwise
+   [config/prod/prow/jobs/config.yaml](../config/prod/prow/jobs/config.yaml), otherwise
    the presubmit test will fail. Create a pull request with the changes. Once
    it's merged the configs will be automatically updated by a postsubmit job.
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
1. Update [config_knative.yaml](../config/prod/prow/config_knative.yaml)
   (usually, copy and update the existing configuration from another
   repository). Run `./hack/generate-configs.sh` to regenerate
   [config/prod/prow/jobs/config.yaml](../config/prod/prow/jobs/config.yaml),
   otherwise the presubmit test will fail. Create a pull request with the
   changes. Once it's merged the configs will be automatically updated by a
   postsubmit job.
```